### PR TITLE
Create Calculator component

### PR DESCRIPTION
**Pull request by oyhpnayiaw**
Create stateless Calculator component using functional component.

Live Demo - https://oyhpnayiaw-as-micronaut.github.io/math-magicians/

# Preview Image 
Screen Shot 2023-02-13 at 6 06 37 PM

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => {
```
Comment (julie-ify) [inline src/components/Calculator.js:4]: - [ ] The components should be class-based. Kindly refactor this to a class component.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,5 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```
Comment (julie-ify) [inline src/App.js:3]: - [ ] The components should be class-based. Kindly refactor this to a class component

File: README.md [hunk 3]
```diff
@@ -54,9 +54,7 @@ React.js
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (julie-ify) [inline README.md:56]: - [ ] Kindly remove the marked files as generated by the boilerplate.

owpq
oas

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @oyhpnayiaw,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (oyhpnayiaw) [reply src/components/Calculator.js:4]: Hello @julie-ify, But in the project requirement it said to use functional component. Can you explain more?

See the image below.
Screen Shot 2023-02-13 at 6 19 54 PM

Comment (oyhpnayiaw) [reply src/components/Calculator.js:4]: nvm, I mistaken with function component. 😄

Comment (julie-ify) [reply src/components/Calculator.js]: @oyhpnayiaw alright.

**Review by julie-ify — APPROVED** [review]
Hi @oyhpnayiaw,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**